### ISSUE TITLE
additional metainformation about repositories

### ISSUE THREAD

**Issue by kamilsk** [kaxap/awesome-go-processed#2]
> Could do you also add some additional metainformation about repositories? I mean last commit' date, last closed issue, etc. Some "live" status.

**Comment by kaxap** [comment]
Great idea! I'll see what can be done when I get home.

**Comment by kamilsk** [comment]
you can try to use github client https://developer.github.com/v3/libraries/ for Python

**Comment by kaxap** [comment]
Added last update time [813f259](https://github.com/kaxap/awesome-go-processed/commit/813f259ffe4c5c0c0eaf6324ce22c21de67b1484). 

To parse last commit and last closed issue I would need to perform additional two API queries which will result in 3x higher request rate, e.g. 5691 requests instead of 1897. I guess Github won't be happy with such many requests everyday.

**Comment by kamilsk** [comment]
> To parse last commit and last closed issue I would need to perform additional two API queries which will result in 3x higher request rate, e.g. 5691 requests instead of 1897

what about https://github.com/tomasbasham/ratelimit or something similar? you can do uniformly distributed load

**Comment by kaxap** [comment]
This doesn't help a lot. Github [returns](https://developer.github.com/v3/rate_limit/) info on rate limits, however after several dozen calls Github suspends API calls for an hour or two. 

Still, AFAIK, update info indicates last changes made to repository including commits and issues.

**Comment by kamilsk** [comment]
I mean you can call GitHub API not so often as you can. For example, GitHub allows you to make 100 RPS from one IP, but you can use limit 50 RPS on your side instead of possible value. It increases the crawl time but you solve the problem

> I guess Github won't be happy with such many requests everyday.

**Comment by kamilsk** [comment]
> I guess Github won't be happy with such many requests everyday.

Also, I think it is not a big deal for GitHub. Most likely they do not even notice such a load 😄

**Comment by kaxap** [comment]
> GitHub allows you to make 100 RPS 
It gives me ~1 hour delay after few dozen requests. Even with 0.5 sec delay before each request.
Maybe I should make authorized requests?

**Comment by kamilsk** [comment]
Yes, you can use your personal token. You can generate it in [settings](https://github.com/settings/tokens). But do not include it in the code, use environment variable instead.

**Comment by kaxap** [comment]
Replaced last update time with last commit time. [3bc10c0](https://github.com/kaxap/awesome-go-processed/commit/3bc10c0cea06e79f8d1ee033acf05ae8715b328a)

Seems like some repos in the list haven't been updated for 6-8 years.

**Comment by kamilsk** [comment]
Good to know. Some repositories have less than five commits, and I don't know why they're on the awesome list.

**Comment by kaxap** [comment]
It is possible to include a total number of commits as well. I have some concerns however, the table might become oversaturated with all kinds of data.

**Comment by phanirithvij** [comment]
@kaxap I think you accidentally gitinored a source file? last_commit.py in https://github.com/kaxap/awesome-go-processed/commit/3bc10c0cea06e79f8d1ee033acf05ae8715b328a

It is not in the repo and thus the repo is unusable.
